java-spring/accounts-query-side-backend/src/main/java/net/chrisrichardson/eventstore/javaexamples/banking/backend/queryside/accounts/AccountQueryWorkflow.java

```diff
@@ -34,7 +35,7 @@ public AccountQueryWorkflow(AccountInfoUpdateService accountInfoUpdateService) {
   public void create(DispatchedEvent<AccountOpenedEvent> de) {
     AccountOpenedEvent event = de.getEvent();
     String id = de.getEntityId();
-    String eventId = de.getEventId().asString();
+    Int128 eventId = de.getEventId();
     logger.info("**************** account version=" + id + ", " + eventId);
```

Comment:
@dartpopikyardo No string concatenation when logging. This should be logger.info("... {} ...{}", param1, param2);